Cargo.toml

```diff
@@ -69,6 +69,7 @@ wasmparser = { workspace = true }
 tracing = { workspace = true }
 log = { workspace = true }
 humantime = { workspace = true }
+tempfile = { workspace = true }
```

Comment:
Let's make this an optional dependency and enable it when the `explore` cargo feature is enabled.

```suggestion
tempfile = { workspace = true }
```

```toml
[features]
# ...
explore = ["dep:wasmtime-explorer", "dep:tempfile"]
#...
```